.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] Hi, you've done well to add your test.md file to the .gitignore file 👏. Although could you kindly add `node_modules` to your .gitignore file as well. doing this will prevent your node_modules folder with all the packages from being committed into repository. Thanks & happy coding.